orange/src/main/java/com/hartwig/hmftools/orange/algo/linx/DnaFusionSelector.java

```diff
@@ -34,6 +35,16 @@ public static List<LinxFusion> selectInterestingUnreportedFusions(@NotNull List<
         return filtered;
     }
 
+    @NotNull
+    public static List<LinxFusion> selectViableSomaticFusions(@NotNull List<LinxFusion> somaticFusions,
```

Comment:
Another minor but maybe add "Additional" to function name? The function name now implies you select all viable fusions, while you only select viable fusions that are not otherwise reported.